test/Spec/MockContract.hs

```diff
@@ -159,46 +175,56 @@ instance Default ContractEnvironment where
     ContractEnvironment
       { cePABConfig = def {pcNetwork = Mainnet}
       , ceContractInstanceId = ContractInstanceId UUID.nil
+      , ceContractState = ContractState <$> unsafePerformIO $ newTVarIO Map.empty
```

Comment:
This is a crime but I'll allow it

Reply:
Yes, I know, I just needed to initialise this for tests, hoping that this is a justifiable use case.